templates/plug_html_template.tmpl

```diff
@@ -0,0 +1,23 @@
+<html>
+	<body style="background-image:url('https://i.imgur.com/YWfFxsh.png');" link="white" vlink="white" alink="white">
+                <center>

```

Comment:
As much as I love `<center>` tags you should take the time to do this properly and avoid using deprecated tags. For more information on that please read https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center